public/javascripts/SVLabel/src/SVLabel/mission/Mission.js

```diff
@@ -355,8 +355,12 @@ function Mission(parameters) {
         }
     }
 
+    function setLabelCounts(labelCounts){
```

Comment:
What is this method? It seems like `labelCoutnsAtCompletion`. If so, please fix.